lyricsgenius/api/base.py

```diff
@@ -21,8 +21,13 @@ def __init__(
         timeout=5,
         sleep_time=0.5
     ):
+        self._session = requests.Session()
```

Comment:
Whenever the tests run, in the end there's a warning that there are two unclosed sockets. This would be okay since we don't close the session (`genius._session.close()`) at the end of the tests, but the problem is that there are two of them which probably means that we've created two sessions. I moved declaring the session to inside the `__init__` since it's only called once, but that the warnings still persisted.
I think this warrants some investigation because we don't want to be opening redundant sessions. But I don't know how is it that two sessions are created since we only execute `self._session = requests.Session()` once.

Reply:
```bash
sys:1: ResourceWarning: unclosed <ssl.SSLSocket fd=4, family=AddressFamily.AF_INET, type=2049, proto=6, laddr=('10.30.255.116', 59942), raddr=('104.18.18.96', 443)>
sys:1: ResourceWarning: unclosed <ssl.SSLSocket fd=3, family=AddressFamily.AF_INET, type=2049, proto=6, laddr=('10.30.255.116', 59940), raddr=('104.18.18.96', 443)>
```